pkg/instancetype/preference/requirements/cpu_test.go

```diff
@@ -0,0 +1,399 @@
+//nolint:lll
```

Comment:
nit - If you have time could you address this in both test files with a separate commit?

Reply:
Done :+1: